lib/code_corps/policy/message.ex

```diff
@@ -0,0 +1,26 @@
+defmodule CodeCorps.Policy.Message do
+  @moduledoc """
+  Handles `User` authorization of actions on `Message` records
+  """
+
+  import CodeCorps.Policy.Helpers, only: [administered_by?: 2, get_project: 1]
+
+  alias CodeCorps.{Message, User}
+
+  def show?(%User{id: user_id}, %{initiated_by: "user", author_id: author_id})
+    when user_id == author_id do
+    true
+  end
+  def show?(%User{} = user, %Message{} = message) do
+    message |> get_project() |> administered_by?(user)
+  end
+  def show?(_, _), do: false
+
+  def create?(%User{id: id}, %{"initiated_by" => "user", "author_id" => author_id}) do
+    id == author_id
+  end
```

Comment:
Only because the fallback is false. I'll make the change, but I'm not sure if I understand the argument.

Reply:
Using a guard may be faster (not sure if it actually is here) and, additionally, stresses that there are no side effects to the check since only kernel functions can be used in guards.

It seems arbitrary, and probably doesn't affect much, if anything here, but generally it's preferred to avoid entering the function at all if it's easy to do, which it is here.